awx/ui_next/src/screens/Template/TemplateList/TemplateListItem.test.jsx

```diff
@@ -161,4 +162,88 @@ describe('<TemplateListItem />', () => {
     );
     expect(wrapper.find('ExclamationTriangleIcon').exists()).toBe(false);
   });
+  test('clicking on template from templates list navigates properly', () => {
+    const history = createMemoryHistory({
+      initialEntries: ['/templates'],
+    });
+    const wrapper = mountWithContexts(
+      <Route
```

Comment:
does this need the <Route> wrapping it for history changes to work? I don't think we've had to do that for similar tests in other components

Reply:
That is how I have done it elsewhere in our project, but further exploration says no, we don't need Route.